app/controllers/document.js

```diff
@@ -26,6 +26,20 @@ export default class DocumentController extends Controller {
     }
   }
 
+  @action
+  tabChanged(tabName){
+    this.activeTab = tabName;
+  }
+
+  @action
+  setSelectedVersion(piece) {
+    this.router.transitionTo(
```

Comment:
`replaceWith` ipv `transitionTo` to stop browser history additions on tab switch